### PR TITLE
Drawer: Make content scroll by default

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- wraps the drawer content in a scrollbar by default
- deprecates the `scrollableContent` prop and marks it for removal in the next version
- removes all uses of `scrollableContent` in the core grafana codebase

**Why do we need this feature?**

- cleaner interface for the `Drawer` component

**Who is this feature for?**

- everyone! 🙌 

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by axelavargas — COMMENTED** [review]
Mr Harrison 🕶️  code looks good to me, good job 🙌🏾 . 

However, I wonder, if we are deprecating the property `scrollableContent` shouldn't we keep the old behavior to be backward compatible?

P.S. We could also update the `Drawer.mdx` to include this default behavior :)

**Comment by ashharrison90** [comment]
> Mr Harrison 🕶️ code looks good to me, good job 🙌🏾 .
> 
> However, I wonder, if we are deprecating the property `scrollableContent` shouldn't we keep the old behavior to be backward compatible?
> 
> P.S. We could also update the `Drawer.mdx` to include this default behavior :)

good shout on restoring the old behaviour. i've done that, just made `scrollableContent` default. 

re: updating the documentation; i'm not sure what i'd write. especially since we're deprecating the prop, we don't really want to encourage anyone to use it... "this component now scrolls exactly as you'd expect it to"? 😅 that's why i find this whole prop so weird. it's not something we should have ever exposed as a configuration option. this is just how the component should work.

**Comment by axelavargas** [comment]
> re: updating the documentation; i'm not sure what i'd write. especially since we're deprecating the prop, we don't really want to encourage anyone to use it... "this component now scrolls exactly as you'd expect it to"? 😅 that's why i find this whole prop so weird. it's not something we should have ever exposed as a configuration option. this is just how the component should work.

That's a good point!

This component now scrolls exactly as you'd expect it to ---> 🤣 this made my day.

File: packages/grafana-ui/src/components/Drawer/Drawer.tsx [hunk 1]
```diff
@@ -41,7 +40,10 @@ export interface Props {
   size?: 'sm' | 'md' | 'lg';
   /** Tabs */
   tabs?: React.ReactNode;
-  /** Set to true if the component rendered within in drawer content has its own scroll */
+  // TODO remove this prop next major version, it doesn't do anything anymore
```
Comment (axelavargas) [inline packages/grafana-ui/src/components/Drawer/Drawer.tsx:43]:
```suggestion
  // TODO remove this prop next major version
```

**Review by axelavargas — APPROVED** [review]

**Pull request merged**